frappe/public/less/navbar.less

```diff
@@ -264,3 +264,60 @@
 .navbar-default .navbar-brand {
 	color: @text-muted;
 }
+
+.dropdown-energy-points .energy-points-icon {
+    height: 40px;
+    font-size: 14px;
+    text-align: center;
+}
+
+.recent-points-list {
+	width: 300px;
+}
+
+.energy-points-notification {
+	font-size: 7px;
+    position: absolute;
```

Comment:
use tabs for indentation...
<img width="206" alt="Screenshot 2019-08-02 at 10 11 26 AM" src="https://user-images.githubusercontent.com/13928957/62344999-0a63e800-b50e-11e9-88d5-c7b8ee9f6602.png">

you can enable `renderWhitespace` setting in your code editor so that you can fix such issues beforehand...
<img width="418" alt="Screenshot 2019-08-02 at 10 07 10 AM" src="https://user-images.githubusercontent.com/13928957/62344880-89a4ec00-b50d-11e9-8fb4-600d15b8efa6.png">